DEVELOPER.md

```diff
@@ -107,8 +107,6 @@ consider adding your user id to the local docker group.
 sudo usermod -a -G docker ${USER}
 newgrp docker
 ```
-## Install golangci
```

Comment:
Why are we deleting this?

Reply:
The download of the golangci-lint is set up in the makefile as: .PHONY: golangci-lint, and it should be able to download the suitable version of the golangci-lint which is different from the README, move it in the README as it is not required and I don't want to maintain the version of golangci-lint in two places